helix-term/src/commands.rs

```diff
@@ -142,6 +142,18 @@ impl<'a> Context<'a> {
     pub fn count(&self) -> usize {
         self.count.map_or(1, |v| v.get())
     }
+
+    /// Waits on all pending jobs, and then tries to flush all pending write
+    /// operations for all documents.
+    pub fn block_try_flush_writes(&mut self) {
```

Comment:
Let's have this return the same result as `compositor::Context::block_try_flush_writes` and ignore it in `suspend` instead. The current code silently ignores errors for any callers

Reply:
thanks, I incorporated this and your other review!